joint_names: [rh_FFJ1, rh_FFJ2, rh_FFJ3, rh_FFJ4, rh_LFJ1, rh_LFJ2, rh_LFJ3, rh_LFJ4, rh_LFJ5, rh_MFJ1, rh_MFJ2, rh_MFJ3, rh_MFJ4, rh_RFJ1, rh_RFJ2, rh_RFJ3, rh_RFJ4, rh_THJ1, rh_THJ2, rh_THJ3, rh_THJ4, rh_THJ5, rh_WRJ1, rh_WRJ2]
grasps:
  open_hand: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  close_hand: [1.5707, 1.5707, 1.5707, 0.0, 1.5707, 1.5707, 1.5707, 0.0, 0.0, 1.5707, 1.5707, 1.5707, 0.0, 1.5707, 1.5707, 1.5707, 0.0, 0.52, 0.61, 0.0, 1.20, 0.17, 0.0, 0.0]
  abduction_grip: [1.256637061, 1.570796327, 0.2967059728, -0.0872664626, 1.134464014, 1.570796327, 0.2792526803, -0.008099086018, 1.326450232, 1.570796327, 1.431169987, -0.03490658504, 1.396263402, 1.570796327, 1.291543646, -0.0872664626, 0.06981317008, 0.7679448709, -0.03490658504, 0.1919862177, 1.186823891, 0.7679448709, -0.6108652382, 0.03490658503988659]
  medium_wrap: [1.570796327, 1.570796327, 1.291543646, -0.1570796327, 1.570796327, 1.570796327, 1.431169987, -0.06981317008, 1.570796327, 1.570796327, 1.396263402, -0.01047865982, 1.570796327, 1.570796327, 1.064650844, -0.05235987756, 0.01745329252, 1.570796327, -0.0872664626, -0.2094395102, 0.1745329252, 0.5934119457, -0.5235987756, 0.06981317007977318]
  palmar_pinch: [0, 0.6283185307, 1.274090354, 0.2617993878, 1.413716694, 1.570796327, 1.396263402, -0.008099086018, 1.221730476, 1.570796327, 1.431169987, -0.03490658504, 1.291543646, 1.570796327, 1.274090354, -0.06981317008, 0.06981317008, 0, -0.4188790205, 0.1047197551, 0.9948376736, 0.6632251158, -0.5759586532, 0.08726646259971647]
  parallel_extension: [0.01745329252, 0.6806784083, 1.326450232, 0.2617993878, 0.01745329252, 0.7330382858, 1.396263402, 0.01745329252, 0.03490658504, 0.837758041, 1.413716694, -0.1570796327, 0.05235987756, 0.9599310886, 1.151917306, -0.1745329252, 0.2443460953, 0, 0.6806784083, 0.2094395102, 1.221730476, 0.06981317008, -0.4886921906, 0.05235987755982989]
  pinch_sphere: [0.03490658504, 0.9599310886, 1.151917306, -0.03490658504, 0.03490658504, 1.012290966, 1.204277184, 0.1745329252, 0.05235987756, 1.239183769, 1.204277184, -0.3490658504, 0.03490658504, 1.256637061, 1.117010721, -0.3490658504, 0.2617993878, 1.186823891, 0.4188790205, 0.1396263402, 1.221730476, -0.1047197551, -0.5585053606, 0.06981317007977318]
  tip_pinch: [0.436332313, 1.570796327, 0.5410520681, 0.2443460953, 1.308996939, 1.570796327, 1.448623279, -0.008099086018, 1.274090354, 1.570796327, 1.466076572, -0.01745329252, 1.186823891, 1.570796327, 1.274090354, -0.06981317008, 0.06981317008, 1.570796327, -0.436332313, -0.1047197551, 0.9250245036, 0.471238898, -0.6283185307, 0.06981317007977318]
  writing_tripod: [0.3490658504, 1.256637061, 0.837758041, -0.2268928028, 0.4886921906, 1.431169987, 1.151917306, 0.2443460953, 0.9250245036, 1.570796327, 1.413716694, -0.2967059728, 1.221730476, 1.570796327, 1.431169987, -0.3316125579, 0.1221730476, 1.082104136, 0.6457718232, 0.1396263402, 0.471238898, -0.3316125579, -0.4886921906, 0.017453292519943295]
